docs_src/microservices/device/profile/Ch-DeviceProfile.md

```diff
@@ -1,6 +1,102 @@
 # Device Profile
 
-![image](EdgeX_DeviceProfile.png)
+The device profile describes a type of device within the EdgeX system. Each
+device managed by a device service has an association with a device profile,
+which defines that device type in terms of the operations which it supports.
 
-The Device Services Device Profile block diagram shows the Device
-Profile Object classes.
+For a full list of device profile fields and their required values see the [Device Profile Reference](./Ch-DeviceProfileRef.md)
+
+Identification
+--------------
+
+The profile contains various identification fields. The `Name` field is required and must be unique in an EdgeX deployment. Other fields are optional - they are not used in EdgeX but may be populated for informational purposes:
```

Comment:
If the `Name` field must be unique, we need to define what happens when EdgeX encounters a duplicate name.  This way, if someone inadvertently does duplicates a name, the error signature can be easily recognized.

Reply:
Are optional fields **absolutely** not used?  By this I mean, these fields in the device profile are never even read by the device service, let alone stored in Core Metadata or in Core Data, and furthermore, are not "visible," e.g. retrievable by some RESTful API somewhere in EdgeX.  If that is the case, then these fields should not even be in the document.

Otherwise, we need to be clearer on how they may or may not be populated and how they are visible.

Keep in mind, any data that enters "EdgeX" can and will be used by someone's "EdgeX."

Reply:
Clarified that the "optional" fields are not used by device services